backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/webservice/schema/types/CSVAcceptingStore.kt

```diff
@@ -1,5 +1,6 @@
 package app.ehrenamtskarte.backend.stores.webservice.schema.types
 
+// TODO 1591 remove this class
```

Comment:
why this class is used in the new mutation if it should be removed?

Reply:
forgot to remove comment